Library/src/Systems/ECommerce/ECommerce.cpp

```diff
@@ -583,8 +584,16 @@ void CheckoutInfoResult::OnResponse(const csp::services::ApiResponseBase* ApiRes
 	if (ApiResponse->GetResponseCode() == csp::services::EResponseCode::ResponseSuccess)
 	{
 		CheckoutInformationResponse->FromJson(Response->GetPayload().GetContent());
-		CheckoutInformation.StoreUrl	= CheckoutInformationResponse->GetCheckoutUrl();
-		CheckoutInformation.CheckoutUrl = CheckoutInformationResponse->GetCheckoutUrl();
+
+		if (CheckoutInformationResponse->HasStoreUrl())
```

Comment:
This is the actual change. Everything else is formatting that my local Git GUI didn't pick up for some reason